stock_account_inventory_force_date/models/stock_inventory.py

```diff
@@ -91,7 +91,7 @@ def _get_inventory_lines_values(self):
                 lines.extend(self._generate_lines_at_date(loc, product_ids))
         else:
             raise ValidationError(_(
-                'Option %s not available when forcing Inventory Date.',
+                'Option %s not available when forcing Inventory Date.' %
```

Comment:
@sergiocorato  Are you sure of the error you have?

_() translation function supports arguments for string formatting like previous code.

Reply:
Mmm, overriding %s python standard function?

Reply:
Ah, took too long to reply :)